.travis.yml

```diff
@@ -56,6 +56,16 @@ script:
       ./scripts/build/docker/run-command.sh -r ${TARGET_ARCH} -s ${PWD} -c "xvfb-run --server-args=$XVFB_ARGS npm test && ./scripts/ci/ensure-staged-sass.sh";
     fi
 
+deploy:
+  provider: script
+  script: if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
+            ./scripts/build/docker/run-command.sh -r ${TARGET_ARCH} -s ${PWD} -c "make publish-aws-s3";
+          else
```

Comment:
I fixed the `else` inconsistency.

> (and given that we invariably run the same command (here make publish-aws-s3) in both the osx and linux sections, I wonder if the repetition could be eliminated? I'm not very familiar myself with the syntax / logic allowed in Travis config files)

Yeah, I know its ugly, but I can't think of anything, apart from adding the command to a global environment variable at the top of the file and re-use that, but sounds ugly as well.